adversarial_workflow/cli.py

```
@@ -2819,6 +2820,98 @@ def list_evaluators() -> int:
+    print_verification_summary(results)
+
+    # Count blocked/broken
+    blocked_count = sum(1 for r in results if r.status in (URLStatus.BLOCKED, URLStatus.BROKEN))
+
+    # Mark document inline if requested
+    if mark_inline and results:
+        marked_document = mark_urls_inline(document, results)
+        if marked_document != document:
+            with open(file_path, "w") as f:
+                f.write(marked_document)
+            print(f"\n   ✅ Updated document with status badges")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)